_data/developer.yml

```diff
@@ -561,7 +561,10 @@ websites:
     img: twilio.png
     tfa:
       - sms
-    doc: https://www.twilio.com/help/faq/twilio-basics/what-happens-when-i-enable-two-factor-authentication
+      - phone
+      - proprietary
+    doc: https://support.twilio.com/hc/en-us/articles/223136307-Enabling-two-factor-authentication-on-your-Twilio-project
+    exception: Software authentication requires the Authy app.
```

Comment:
```suggestion
```
Listing the 2FA method as "proprietary" is sufficient, we don't list the actual app in these cases.

Reply:
I guess we should remove this exception from Sendgrid then. Am I right?

Reply:
Correct!